### DISCUSSION TITLE
Is is possible to simulate a 3D time dependent lattice scattering?

### DISCUSSION THREAD

**Discussion by Obenprint1** [quantum-visualizations/qmsolve#26] [Q&A]
Hi
I saw from the examples provided a 2D scattering of a single particle in a 'grid' potential, but I'm really interested of doing the same but in 3D. Is there any possibility for that? 
I tried it on my own, but the split-step method isn't inplemented for a 3D time dependent simulation, so I switched to the crank-nicholson method, but I ran in many difficulties visualizing the results as it is not implemented in 3D. I nonetheless tried to write the amplitude of the WF to an external file and visualize myself with a different program, but no luck, the output data gives unphysical results (no propagation of the WF and some jumps between different timesteps). Is it in theory at least, possible to use the 3D CN method for 3D lattice scattering or should even more be implemented in the code (other the visualization)?
Thank you in advance!

**Comment by marl0ny** [comment]
It should definitely be possible - we were actually planning to fully implement the 3D time dependent simulations with visualizations, but this has yet to come to fruition due to lack of time (most of the work done for this project was during lockdowns). 

Anyways, it's possible your issue may have to do with using a momentum value that's higher than what the spatial discretization used can actually resolve, basically [sampling errors](https://en.wikipedia.org/wiki/Aliasing). If that's the case then you can try decreasing the initial momentum or increasing the grid size.
